packages/mobile/src/account/DataSaver.tsx

```diff
@@ -82,16 +99,36 @@ export class DataSaver extends React.Component<Props, State> {
         >
           <Text style={fontStyles.body}>{t('enableDataSaver')}</Text>
         </SettingsSwitchItem>
-        <Modal isVisible={this.state.modalVisible}>
+        <Modal isVisible={this.state.switchOffModalVisible}>
```

Comment:
Could this be one modal with varying content to avoid this duplication?

Reply:
It's not only the content, also callbacks. I think bunch of `()?:` would make it ugly.